src/app/portlets/content-types/form/content-types-form.component.ts

```diff
@@ -132,7 +132,7 @@ export class ContentTypesFormComponent implements OnInit {
      * @memberof ContentTypesFormComponent
      */
     submitForm(): void {
-        if (this.form.valid) {
+        if (this.form.valid && this.canSave) {
```

Comment:
We just need this.canSave here because this check this.form.valid

https://github.com/dotCMS/core-web/blob/54399744abcf74ae95330da9eba988f255043281/src/app/portlets/content-types/form/content-types-form.component.ts#L150